bors.toml

```diff
@@ -1,5 +1,9 @@
 status = [
-  "continuous-integration/travis-ci/push",
+  "systemtest-latenodes",
+  "systemtest-blocks-add-node",
+  "systemtest-hare-mining",
+  "systemtest-sync-blocks",
+  "systemtest-genesis-p2p",
```

Comment:
I'm still not liking this - can't we somehow say that all CI tests must pass without specifying them individually?

Reply:
As I [wrote earlier](https://github.com/spacemeshos/go-spacemesh/pull/2161#issuecomment-714755145), I can't think of an easy way to do this. Based on the [bors documentation](https://bors.tech/documentation/#configuration-borstoml) and https://github.com/bors-ng/bors-ng/issues/730, the `bors.toml` file needs a "List of commit statuses" that matches what GitHub displays as "Required statuses" (see below 👇). If we want the tests to all run in parallel, each needs to be an independent "job" (in the lingo of [Workflow syntax for GitHub Actions](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#jobs)) which means it'll get a separate line under "Required statuses."

I can only think of two ways to do this. 1. Hack bors itself, ideally so that it gets the list of "Required statuses" itself using the GitHub API, or 2. write a Makefile script that auto-generates `bors.toml` somehow, although as I indicated earlier it might still take some complicated finagling to get bors to read a dynamically-generated config file. I suggest that we leave this as a separate issue and a task for later - it would be a great, self-contained task for a community contribution.

Reply:
Opened #2172 to follow up